i18n/en.pot

```diff
@@ -182,6 +182,9 @@ msgstr ""
 msgid "Aggregation method"
 msgstr ""
 
+msgid "Maximum number of groups are {{maxBands}}"
```

Comment:
I suggest wording this `Earth engine layers can’t have more than {{maxBands}} groups.`.